elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/FilterIT.groovy

```diff
@@ -1407,187 +1428,214 @@ class FilterIT extends AbstractIntegrationTestInitializer {
 
         /* Test RSQL Global */
-        result = mapper.readTree(RestAssured.get("book?filter=authors.name=='Null Ned';title=='Life with Null Ned'").asString());
-        Assert.assertEquals(result.get("data").size(), 1);
-        Assert.assertEquals(result.get("data").get(0).get("attributes").get("title").asText(), "Life with Null Ned");
-        Assert.assertEquals(result.get("data").get(0).get("relationships").get("authors").get("data").get(0).get("id").asText(), nullNedId);
+        result = mapper.readTree(RestAssured.get("book?filter=authors.name=='Null Ned';title=='Life with Null Ned'").asString())
+        Assert.assertEquals(result.get("data").size(), 1)
+        Assert.assertEquals(result.get("data").get(0).get("attributes").get("title").asText(), "Life with Null Ned")
+        Assert.assertEquals(result.get("data").get(0).get("relationships").get("authors").get("data").get(0).get("id").asText(), nullNedId)
     }
 
     @Test
-    public void testFilterAuthorsByBookChapterTitle() {
+    void testFilterAuthorsByBookChapterTitle() {
         /* Test Default */
-        def result = mapper.readTree(RestAssured.get("author?filter[author.books.chapters.title][in]=Viva la Roma!,Mamma mia I wantz some pizza!").asString());
-        Assert.assertEquals(result.get("data").size(), 2);
-        String last = null;
+        def result = mapper.readTree(RestAssured.get("/author?filter[author.books.chapters.title][in]=Viva la Roma!,Mamma mia I wantz some pizza!").asString())
+        Assert.assertEquals(result.get("data").size(), 2)
         for (JsonNode author : result.get("data")) {
-            String name = author.get("attributes").get("name").asText();
-            Assert.assertTrue(("Isaac Asimov".equals(name) || "Null Ned".equals(name)) && !name.equals(last));
+            String name = author.get("attributes").get("name").asText()
+            Assert.assertTrue(name == "Isaac Asimov" || name == "Null Ned")
         }
 
         /* Test RSQL Global */
-        result = mapper.readTree(RestAssured.get("author?filter=books.chapters.title=in=('Viva la Roma!','Mamma mia I wantz some pizza!')").asString());
-        Assert.assertEquals(result.get("data").size(), 2);
-        last = null;
+        result = mapper.readTree(RestAssured.get("/author?filter=books.chapters.title=in=('Viva la Roma!','Mamma mia I wantz some pizza!')").asString())
+        Assert.assertEquals(result.get("data").size(), 2)
         for (JsonNode author : result.get("data")) {
-            String name = author.get("attributes").get("name").asText();
-            Assert.assertTrue(("Isaac Asimov".equals(name) || "Null Ned".equals(name)) && !name.equals(last));
+            String name = author.get("attributes").get("name").asText()
+            Assert.assertTrue(name == "Isaac Asimov" || name == "Null Ned")
         }
     }
 
     @Test
-    public void testGetBadRelationshipRoot() {
+    void testFilterAuthorBookByPublisher() {
```

Comment:
There are tests for both. I have the default filter test on top followed by the rsql test, like the other tests here.